pkgs/ffigen/test/native_objc_test/protocol_test.m

```diff
@@ -13,11 +13,15 @@
   int32_t y;
 } SomeStruct;
 
-@protocol MyProtocol<NSObject>
+@protocol SuperProtocol<NSObject>
 
 @required
 - (NSString*)instanceMethod:(NSString*)s withDouble:(double)x;
 
+@end
+
+@protocol MyProtocol<SuperProtocol>
```

Comment:
Should we also add an integration test for an existing API? For example the one you mentioned? https://developer.apple.com/documentation/foundation/nsurlsessiontaskdelegate/1411626-urlsession

Reply:
Yep. Brian suggests testing *all* of them 🚀 
https://github.com/dart-lang/native/issues/1183